### PR TITLE
layouts for app and index feature for tasks

### PR DESCRIPTION
**Pull request by duccs-0007**
layouts for app and index feature for tasks

### CODE REVIEW

**Comment by duccs-0007** [comment]
ready

File: resources/lang/en/tasks.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+
+    'lb_name'   => 'Task',
+    'sb_add'   => 'Add Task'
```
Comment (tranducquoc) [inline resources/lang/en/tasks.php]: 1 space thôi chứ

**Comment by duccs-0007** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**